### PR TITLE
Add clipboard support to the RSS feed link

### PR DESCRIPTION
**Pull request by nickvergessen**
activity_-_nextcloud_-_2016-09-22_18 21 26

@jancborchardt want to review?

### CODE REVIEW

File: templates/stream.app.navigation.php [hunk 1]
```diff
@@ -45,7 +45,11 @@
 		<div id="app-settings-content">
 			<input type="checkbox"<?php if ($_['rssLink']): ?> checked="checked"<?php endif; ?> id="enable_rss" class="checkbox" />
 			<label for="enable_rss"><?php p($l->t('Enable RSS feed'));?></label>
-			<input id="rssurl"<?php if (!$_['rssLink']): ?> class="hidden"<?php endif; ?> type="text" readonly="readonly" value="<?php p($_['rssLink']); ?>" />
+
+			<span id="rssurl"<?php if (!$_['rssLink']): ?> class="hidden"<?php endif; ?>>
+				<input class="feed-link" type="text" readonly="readonly" value="<?php p($_['rssLink']); ?>" />
+				<a class="clipboardButton icon icon-clippy" data-clipboard-target="#rssurl input"></a>
```
Comment (jancborchardt) [inline templates/stream.app.navigation.php]: No need for the separate `icon` class here. :) Also, why use an additional class »clipboardButton« when you can simply use icon-clippy? And for classes please only use dashes, never camel case!

Comment (nickvergessen) [inline templates/stream.app.navigation.php]: This was all copy pasted ;)

But sure I can address this.

Comment (jancborchardt) [inline templates/stream.app.navigation.php]: That’s what I thought :D so we should fix it going forward that not more people copypaste useless code. ;)

Comment (nickvergessen) [inline templates/stream.app.navigation.php]: Updated

**Comment by jancborchardt** [comment]
Seems to work :+1:

**Pull request merged**